### ISSUE TITLE
[crash] Something went wrong

### ISSUE THREAD

**Issue by nguyenhoangnam3111996** [kubeshop/monokle#1811]
**Describe the bug**
OS: ubuntu 20.04

**Steps to reproduce**

**Stacktrace** 

 ```
Cannot set properties of undefined (setting 'cluster')
The error may be correlated with this previous error:
TypeError: Cannot set properties of undefined (setting 'cluster')
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:334304:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:334273:23)
    at mergeConfigs (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:61392:237)
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:61033:3744
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60956:25
    at memoized (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60864:20)
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60970:40
    at Object.memoized [as current] (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60864:20)
    at Object.checkForUpdates [as onStateChange] (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:635653:48)


 TypeError: Cannot set properties of undefined (setting 'cluster')
The error may be correlated with this previous error:
TypeError: Cannot set properties of undefined (setting 'cluster')
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:334304:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:334273:23)
    at mergeConfigs (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:61392:237)
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:61033:3744
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60956:25
    at memoized (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60864:20)
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60970:40
    at Object.memoized [as current] (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60864:20)
    at Object.checkForUpdates [as onStateChange] (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:635653:48)


    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:334304:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:334273:23)
    at mergeConfigs (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:61392:237)
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:61033:3744
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60956:25
    at memoized (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60864:20)
    at file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60970:40
    at memoized (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:60864:20)
    at useSelectorWithStoreAndSubscription (file:///tmp/.mount_MonoklW7iGa6/resources/app.asar/build/bundle.main.js:635620:30)
```

**Comment by devcatalin** [comment]
Hi @nguyenhoangnam3111996 , thank you for this crash report!
Could you tell us when has this happened or if you have a rough idea on how we can reproduce it?

**Comment by nguyenhoangnam3111996** [comment]
it happened when i tried to open folder with many subfolders in it

**Comment by devcatalin** [comment]
Hi @nguyenhoangnam3111996, we released the `1.7.5` version of Monokle which includes a fix for this error.
You can download it here: https://github.com/kubeshop/monokle/releases/tag/v1.7.5
I'm closing this issue but please let us know if the problem persists after 1.7.5.
Thank you!
